system-tests/tests/src/testFixtures/java/org/eclipse/dataspaceconnector/system/tests/utils/TransferSimulationUtils.java

```diff
@@ -185,7 +185,7 @@ private static ChainBuilder waitForTransferCompletion() {
         return group("Wait for transfer")
                 .on(exec(session -> session.set("status", "INITIAL"))
                         .doWhileDuring(session -> transferNotCompleted(session),
-                                Duration.ofSeconds(30))
+                                Duration.ofSeconds(60))
```

Comment:
- Given the ADF SLA, this is probably insufficient to make the test stable?
- Why change the setting also for non-ADF test?

We could add methods to (a renamed) `TransferRequestFactory` to make this parameterizable